pkg/scheduler/framework/plugins/podtopologyspread/scoring.go

```diff
@@ -166,8 +166,9 @@ func (pl *PodTopologySpread) PreScore(
 		}
 
 		for i, c := range state.Constraints {
-			if pl.enableNodeInclusionPolicyInPodTopologySpread &&
-				!c.matchNodeInclusionPolicies(pod, node, requiredNodeAffinity) {
+			// Ignore parsing errors for backwards compatibility.
```

Comment:
```suggestion
			// Error from this function is a parsing error for `Gt` `Lt` tolerations or taints used with them. 
			// We can just regard them as unmatched because that is how the tainttoleration plugin handles them.
```